1-mandatory/2-quotegenerator/index.html

```diff
@@ -11,6 +11,12 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <section id=" book">
+      <p id="quote"></p>
+      <p id="author"></p>
+      <button id="click-button"> New quote</button>
```

Comment:
You can give an event handler here.  Then in the js file implement this by building out from smaller functions. 

`<button onclick="showRandomQuote()" id="click-button"> New quote</button>`